src/openapi/streaming/streaming.js

```diff
@@ -523,6 +525,103 @@ Streaming.prototype.disposeSubscription = function(subscription) {
     }
 };
 
+function handleSubscriptionReadyForUnsubscribe(subscriptions, resolve) {

```

Comment:
Uber nitpick. What do you think about moving those new private functions above Streaming constructor. It seems like a general structure we use for this file (private methods and then constructor and public ones). It makes it more consistent, i think.